container/dockershell.sh

```diff
@@ -57,6 +57,7 @@ OPTIONS:
                   e.g. "db2eventstore-1578174815082"
 --user         User name of the Event Store server
 --password     Password of the Event Store server
+--es-version   Event Store target cluster version used to tag image
```

Comment:
Should add that if not specified we will assume the "latest" image tag which may not be correct if you built with an earlier release.

Reply:
think we should be requiring version and relying on a defult version